CMakeLists.txt

```diff
@@ -276,7 +276,7 @@ if (APPLE)
 endif (APPLE)
 
 # Set up DEBUG define
-set_directory_properties(PROPERTIES COMPILE_DEFINITIONS_DEBUG DEBUG=1)
+set_directory_properties(PROPERTIES COMPILE_DEFINITIONS $<$<CONFIG:Debug>:DEBUG=1>)
```

Comment:
This line used to do nothing. My 'fix' enabled it, and that's what's killing stuff.